package.json

```diff
@@ -16,5 +16,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "eslint": "^5.4.0",
+    "eslint-config-google": "^0.9.1"
```

Comment:
Curious on preference for eslint google. I tend to like just the recommended because google and others can have strictness that is unnecessary for a smaller project in my opinion.

Reply:
No real preference, just saw a choice between google, airbnb, and recommended and just went with google for no real reason :P